### PR TITLE
Atp develop

### PR DESCRIPTION
**Pull request by andrewtpoe**
- Program takes user name, does not allow blank entry.
- Program asks user for menu option selection concerning legal standing. Does not allow invalid selection.
- Program returns smart aleck saying from assignment page...

### CODE REVIEW

File: assignment.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# 1.1 User Input Assignment
+
+puts "Hello. Please tell me your name."
+
+name_LCV = 1
+while name_LCV == 1
+  print ">> "
+  name = gets.chomp
+  print ""
```
Comment (rpearce) [inline assignment.rb:9]: What is this extra `print` doing? I'm not sure it is doing anything noticeable, so if it's not supposed to be there, you could probably remove it.

File: assignment.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+\t2. I make sure others obey the law.
+\t3. I live outside the law.
+\t4. Questionable.}
+puts "Please enter a selection from the menu above."
+
+decision_LCV = 1
+while decision_LCV == 1
+  print ">> "
+  decision = gets.chomp
+  print ""
```
Comment (rpearce) [inline assignment.rb:32]: same comment as above

**Comment by rpearce** [comment]
:+1:
Go ahead and close/merge this.

**Comment by andrewtpoe** [comment]
That print statement is doing nothing. My bad.

**Pull request merged**